core/App/components/misc/QRScanner.tsx

```diff
@@ -103,11 +113,21 @@ const QRScanner: React.FC<Props> = ({ handleCodeScan, error, enableCameraOnError
         <CameraViewContainer portrait={portraitMode}>
           <QRScannerClose onPress={() => navigation.goBack()}></QRScannerClose>
           <View style={styles.errorContainer}>
-            {error ? (
-              <>
-                <Icon style={styles.icon} name="cancel" size={30}></Icon>
-                <Text style={[TextTheme.caption, { color: ColorPallet.grayscale.white }]}>{error.message}</Text>
-              </>
+            {popupVisable && error ? (
```

Comment:
Was this a requested change?

Reply:
yes, in this issue under the `Scan Errors` link: https://github.com/bcgov/bc-wallet-mobile/issues/481

Reply:
Can we split this out since it's a separate fix from refactoring the popups?

Reply:
sure

Reply:
okay I split the change